librocksdb_sys/src/lib.rs

```diff
@@ -214,6 +214,13 @@ pub unsafe fn new_lru_cache(opt: *mut DBLRUCacheOptions) -> *mut DBCache {
     crocksdb_cache_create_lru(opt)
 }
 
+/// # Safety
+/// 
+/// `DBCache` should be pointed to a valid Cache instance. 
+pub unsafe fn lru_cache_set_capacity(cache: *mut DBCache, capacity: usize) {
```

Comment:
You should add a member function to `Cache`.

Reply:
done